yb_migrate/cmd/exportData.go

```diff
@@ -94,14 +97,14 @@ func exportDataOffline() bool {
 				} else if len(parts) == 2 {
 					tableList = append(tableList, table)
 				} else {
-					fmt.Println("invalid value for --table-list flag")
-					os.Exit(1)
+					utils.ErrExit("invalid value for --table-list flag")
 				}
 			}
 		} else {
 			tableList = userTableList
 		}
 
+		log.Infof(fmt.Sprintf("table list flag values for data export: %v\n", tableList))
```

Comment:
No trailing `\n`.
No need of `fmt.Sprintf()`.
Be concise and complete. `table list for data export: %v`.

Fix all such occurrences.